src/main/java/com/coveo/configuration/parameterstore/strategy/MultiRegionParameterStorePropertySourceConfigurationStrategy.java

```diff
@@ -35,23 +35,29 @@ public void configureParameterStorePropertySources(ConfigurableEnvironment envir
         String lastRegion = regions.get(0);
 
         // We only want to halt boot (if true) for the last region
-        environment.getPropertySources().addFirst(buildParameterStorePropertySource(lastRegion, haltBoot));
+        environment.getPropertySources()
+                   .addFirst(buildParameterStorePropertySource(ssmClientBuilder, lastRegion, haltBoot));
 
         regions.stream()
                .skip(1)
                .forEach(region -> environment.getPropertySources()
-                                             .addFirst(buildParameterStorePropertySource(region, false)));
+                                             .addFirst(buildParameterStorePropertySource(ssmClientBuilder,
+                                                                                         region,
+                                                                                         false)));
     }
 
-    private ParameterStorePropertySource buildParameterStorePropertySource(String region, boolean haltBoot)
+    private ParameterStorePropertySource buildParameterStorePropertySource(AWSSimpleSystemsManagementClientBuilder ssmClientBuilder,
```

Comment:
all those formatting changes are normal?

Reply:
yup

Reply:
dunno why github does not highlight the change but there is a parameter added